float32 robot_electricity
int32 robot_state
int32 robot_driver_state
int32 robot_camera_state


